src/Plugin.php

```diff
@@ -14,10 +14,82 @@
 namespace DebugKit;
 
 use Cake\Core\BasePlugin;
+use Cake\Core\Configure;
+use Cake\Core\PluginApplicationInterface;
+use Cake\Event\EventManager;
+use Cake\Http\MiddlewareQueue;
+use DebugKit\Middleware\DebugKitMiddleware;
+use DebugKit\Panel\DeprecationsPanel;
 
 /**
- * Plugin class for CakePHP 3.6.0 plugin collection.
+ * Plugin class for CakePHP plugin collection.
  */
 class Plugin extends BasePlugin
 {
+    /**
+     * @var \DebugKit\ToolbarService
+     */
+    protected $service;
+
+    /**
+     * Load all the application configuration and bootstrap logic.
+     *
+     * @param \Cake\Core\PluginApplicationInterface $app The host application
+     * @return void
+     */
+    public function bootstrap(PluginApplicationInterface $app): void
+    {
+        $service = new ToolbarService(EventManager::instance(), (array)Configure::read('DebugKit'));
+        $this->service = $service;
+
+        if (!$service->isEnabled() || php_sapi_name() === 'cli' || php_sapi_name() === 'phpdbg') {
+            return;
+        }
+
+        $this->setDeprecationHandler($service);
+
+        if (!$this->isEnabled('routes')) {
```

Comment:
This is not necessary, there's already a routes() hook method which will handle routes file inclusion.

Reply:
Remove it then?

Reply:
Yes